sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/client_base.py

```diff
@@ -55,6 +70,25 @@ def _create_config(credential, api_version=None, **kwargs):
 
         return config
 
+    def _create_client(self, credential, pipeline, api_version):
+        # type: (TokenCredential, Pipeline, str) -> KeyVaultClient
```

Comment:
Same mypy issue as with `KeyVaultClientConfiguration` but harder to fudge because `KeyVaultClient` implementations don't share an unversioned base class. You could import both when type checking and use a `Union` ([for example](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/azure/identity/_credentials/environment.py#L18-L22)).